codespell_lib/data/dictionary.txt

```diff
@@ -11674,7 +11674,7 @@ iterm->term, intern,
 iterpreter->interpreter
 iterrupt->interrupt
 iteself->itself
-ith->with
+ith->with, disabled because of ordinal form of i, like nth
```

Comment:
travis doesn't like that a 2nd comma was used:  
```
line       = b'ith->with, disabled because of ordinal form of i, like nth\n'
msg        = 'error %s: correction %r contains a comma *not* followed by a space'
```

Suggested edit would be `ith->with, disabled because of ordinal form of 'i' (like 'nth')`

Reply:
It's good that it caught that! Now changed